.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ on:
       - 'APNS/**'
       - 'PusherMainView/**'
       - 'pusher/**'
-  pull_request:
+  pull_request_target:
```

Comment:
@apparition47 Do you think it's worth to set `permissions:` just in case?

Reply:
Yes, but I can't figure out the combo of permissions this workflow needs. [It seems to use the GH upload artifact API](https://github.com/kishikawakatsumi/xcresulttool/blob/main/src/main.ts#L128) which I think requires the `actions:write` permission but I still can't get it to work.

Reply:
Hmm. Did you try setting `permissions: write-all` temporarily to see if that helps?

Reply:
Doesn't `pull_request_target` already have `write-all` permissions?

Reply:
Hmm, maybe. I'm not sure

Reply:
I think I got it working with `checks:write`

Reply:
Great!